velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -144,5 +202,142 @@ TEST_F(ContainerRowSerdeTest, nested) {
   testRoundTrip(nestedArray);
 }
 
+TEST_F(ContainerRowSerdeTest, compareNullsInArrayVector) {
+  auto data = makeArrayVector<int64_t>({
+      {1, 2},
+      {1, 5},
+      {1, 3, 5},
+      {1, 2, 3, 4},
+  });
+
+  std::vector<HashStringAllocator::Position> positions;
+  auto byteStreams = serialize(data, positions);
+  auto arrayVector = makeNullableArrayVector<int64_t>({
+      {1, 2},
+      {1, 3},
+      {1, 5},
+      {std::nullopt, 1},
+  });
+  SelectivityVector rows(arrayVector->size());
+  DecodedVector decodedVector;
+  decodedVector.decode(*arrayVector, rows);
+
+  CompareFlags compareFlags{
+      true, // nullsFirst
+      true, // ascending
+      false, // equalsOnly
+      CompareFlags::NullHandlingMode::StopAtNull};
+  testCompareWithNulls(
+      decodedVector,
+      positions,
+      byteStreams,
+      {0, 1, -1},
+      std::nullopt,
+      compareFlags);
```

Comment:
Since we only test different values of equalsOnly and nullHandlingMode, the test may be more readable if we change testCompareWithNulls to take these 2 values instead of CompareFlags.

Reply:
Fixed